cvat-ui/src/components/annotation-page/canvas/brush-tools.tsx

```diff
@@ -0,0 +1,279 @@
+// Copyright (C) 2022 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import './brush-toolbox-styles.scss';
+
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import Button from 'antd/lib/button';
+import Icon, { VerticalAlignBottomOutlined } from '@ant-design/icons';
+import InputNumber from 'antd/lib/input-number';
+import Select from 'antd/lib/select';
+import message from 'antd/lib/message';
+
+import { getCore } from 'cvat-core-wrapper';
+import { Canvas, CanvasMode } from 'cvat-canvas-wrapper';
+import {
+    BrushIcon, EraserIcon, PolygonMinusIcon, PolygonPlusIcon,
+    PlusIcon, CheckIcon, MoveIcon,
+} from 'icons';
+import CVATTooltip from 'components/common/cvat-tooltip';
+import { CombinedState, ObjectType, ShapeType } from 'reducers';
+import LabelSelector from 'components/label-selector/label-selector';
+import { rememberObject, updateCanvasBrushTools } from 'actions/annotation-actions';
+import useDraggable from './draggable-hoc';
+
+const DraggableArea = (
+    <div className='cvat-brush-tools-draggable-area'>
+        <Icon component={MoveIcon} />
+    </div>
+);
+
+const MIN_BRUSH_SIZE = 1;
+let polygonFinishingTipShown = false;
```

Comment:
`useState` for this tip?

Reply:
``useState`` is not necessary here, because it will be shown once the CVAT application is started. 
From my opinion, we should not show the tip again if for example a user reopened the job.  Don't you think so?